File: java/bypstest-api/src/com/wilutions/byps/test/api/inherit/BioLemonService.java
package com.wilutions.byps.test.api.inherit;

import com.wilutions.byps.RemoteException;

/**
 * 
 * @BClientRemote
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 */
public interface BioLemonService extends BioFruitService, LemonService {

  int useParing() throws RemoteException;
}

File: csharp/bypstest-ser/src-ser/byps/test/api/inherit/BioLemonService.cs
﻿using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.inherit
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface BioLemonService : BRemote, 
		BioFruitService, 
		LemonService {
	
		int UseParing();
		void UseParing(BAsyncResult<int> asyncResult) ;
		Task<int> UseParingAsync();
		
		
	}
}  // end namespace
